packages/server-wallet/src/wallet/store.ts

```diff
@@ -200,37 +200,22 @@ export class Store {
     channel = await timer('adding MY state', async () => this.addMyState(channel, signedState, tx));
 
     const sender = channel.participants[channel.myIndex].participantId;
-    let data: Payload = {
-      signedStates: [signedState],
-      objectives: [],
-    };
-    /** todo:
-     * What happens if Bob is adding his signature to prefund0 from Alice?
-     * In this case Bob will send an objective to Alice
-     */
-
-    if (signedState.turnNum === 0) {
-      data = {
-        ...data,
-        objectives: [
-          {
-            participants: channel.participants,
-            type: 'OpenChannel',
-            data: {
-              targetChannelId: channel.channelId,
-              fundingStrategy: 'Direct',
-            },
-          },
-        ],
```

Comment:
What's the reason for removing this objective here?

Reply:
We do not have access to `fundingStrategy` within the `signState` method, but we do within `createChannel`. Also, this bit of code here is within a conditional expression that checks if `turnNum === 0`, which is hyper specific and is only needed within the context of `createChannel`. So the objective gets created and sent within `createChannel`.